src/Misc/Factory/DateTimeFactory.php

```diff
@@ -54,7 +54,7 @@ private function hasDateTimeReportedErrors(): bool
     {
         $errors = DateTimeImmutable::getLastErrors();
         if ($errors === false) {
```

Comment:
I would check if `$errors` is an array to keep backward compatibility
```
if (!is_array($errors)) {
    return false;
}
```

Reply:
This is not needed, it is compatible with PHP <8.2 and 8.2. To quote the documentation: "Before PHP 8.2.0, this function did not return false when there were no warnings or errors. Instead, it would always return the documented array structure." - so, this IF structure for PHP <8.2 will never execute .

Reply:
ok, thanks for PR:)